views/search.mustache

```diff
@@ -11,18 +11,12 @@
       <button type='submit'>Search!</button>
     </form>
     <h1>Snippets on their way!</h1>
-    {{#searchResults}}
-    <p>{{title}}</p>
-    <p>{{codeBody}}</p>
-    <p>{{language}}</p>
-    <p>{{notes}}</p>
-    <p>{{tag}}</p>
-    <hr>
-    {{/searchResults}}
-    <hr>
+    
```

Comment:
It's not visible in this PR, but your `<form>` element's `action` attribute needs fixing. Attributes should basically look like, `x="y"`, or more descriptively, `attribute-name="attribute value"`, e.g. `action="/search"`